 2007-07-16  Jakub Jelinek  <jakub@redhat.com>

	* locales/ar_SA (d_t_fmt, d_fmt, t_fmt, t_fmt_ampm): Replace %.1d
	with supported %e and %.1H with %k.
	* locales/dz_BT (t_fmt): Remove whitespace between % and H.

===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/dz_BT,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/localedata/locales/dz_BT	2006/07/30 22:19:44	1.2
+++ libc/localedata/locales/dz_BT	2007/07/19 17:07:05	1.3
@@ -1293,7 +1293,7 @@
 
 % Appropriate time representation  "%H:%M:%S"
 t_fmt        "<U0F46><U0F74><U0F0B><U0F5A><U0F7C><U0F51>/
-%   H<U0F40><U0F66><U0F62><U0F0B><U0F58>/
+%H<U0F40><U0F66><U0F62><U0F0B><U0F58>/
 %M<U0F40><U0F66><U0F62><U0F0B><U0F46>%S"
 
 % AM/PM signs
===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/ar_SA,v
retrieving revision 1.12
retrieving revision 1.13
diff -u -r1.12 -r1.13
--- libc/localedata/locales/ar_SA	2006/07/30 22:19:44	1.12
+++ libc/localedata/locales/ar_SA	2007/07/19 17:06:52	1.13
@@ -294,16 +294,16 @@
 
 
 % Set up the LC_TIME category
-% d_t_fmt	"%A %.1d %B %Y  %.1H:%M:%S"
-% d_fmt	"%A %.1d %B %Y"
-% t_fmt	"%.1H:%M:%S"
-% t_fmt_ampm	"%.1H:%M:%S"
+% d_t_fmt	"%A %e %B %Y  %k:%M:%S"
+% d_fmt	"%A %e %B %Y"
+% t_fmt	"%k:%M:%S"
+% t_fmt_ampm	"%k:%M:%S"
 
 LC_TIME
-d_t_fmt	"<U0025><U0041><U0020><U0025><U002E><U0031><U0064><U0020><U0025><U0042><U0020><U0025><U0059><U0020><U0020><U0025><U002E><U0031><U0048><U003A><U0025><U004D><U003A><U0025><U0053>"
-d_fmt	"<U0025><U0041><U0020><U0025><U002E><U0031><U0064><U0020><U0025><U0042><U0020><U0025><U0059>"
-t_fmt	"<U0025><U002E><U0031><U0048><U003A><U0025><U004D><U003A><U0025><U0053>"
-t_fmt_ampm	"<U0025><U002E><U0031><U0048><U003A><U0025><U004D><U003A><U0025><U0053>"
+d_t_fmt	"<U0025><U0041><U0020><U0025><U0065><U0020><U0025><U0042><U0020><U0025><U0059><U0020><U0020><U0025><U006B><U003A><U0025><U004D><U003A><U0025><U0053>"
+d_fmt	"<U0025><U0041><U0020><U0025><U0065><U0020><U0025><U0042><U0020><U0025><U0059>"
+t_fmt	"<U0025><U006B><U003A><U0025><U004D><U003A><U0025><U0053>"
+t_fmt_ampm	"<U0025><U006B><U003A><U0025><U004D><U003A><U0025><U0053>"
 day	"<U0627><U0644><U0623><U062D><U062F>"; /
 	"<U0627><U0644><U0625><U062B><U0646><U064A><U0646>"; /
 	"<U0627><U0644><U062B><U0644><U0627><U062B><U0627><U0621>"; /
